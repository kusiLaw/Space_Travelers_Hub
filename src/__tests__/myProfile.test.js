/* eslint-disable no-undef */
import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MyProfile from '../Components/Pages/myProfile';
import store from '../Redux/configureStore';

it('Testing my profile component:', () => {
  const tree = TestRenderer.create(
    <Provider store={store}>
      <MyProfile />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
