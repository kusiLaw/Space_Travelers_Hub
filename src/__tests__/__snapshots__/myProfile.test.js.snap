// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing my profile component: 1`] = `
<div
  className="profile_container"
>
  <div
    className="reserved_container"
  >
    <h2>
      My Missions
    </h2>
    <ul />
  </div>
  <div
    className="reserved_container"
  >
    <h2>
      My Rockets
    </h2>
    <ul />
  </div>
  <div
    className="reserved_container"
  >
    <h2>
      My Dragons
    </h2>
    <ul />
  </div>
</div>
`;
